include/deal.II/grid/grid_tools.h

```diff
@@ -3140,6 +3149,15 @@ namespace GridTools
                   const unsigned int (&)[GeometryInfo<3>::vertices_per_cell]);
 
 
+  template <int dim>
+  double
+  cell_measure(const std::vector<Point<dim>> &  all_vertices,
+               const std::vector<unsigned int> &vertex_indices)
+  {
+    AssertDimension(vertex_indices.size(),
+                    GeometryInfo<dim>::vertices_per_cell);
+    return cell_measure<dim>(all_vertices, vertex_indices.data());
```

Comment:
I am wondering how it finds the right specialization of the function? See:

https://github.com/dealii/dealii/blob/48733b41583ee3fe8f89e2a07f0225f687746587/include/deal.II/grid/grid_tools.h#L3135-L3149

May I ask that you add a test!?